cityofaustin/atd-moped
# makes audit field timestamps match

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/16047

this pr sets `now()` as the default value for all instances of `created_at` and `updated_at` fields in `moped_proj_` tables and refactors the audit field update trigger to only update the `updated_at` field on `UPDATE` (and not on `INSERT`)

## Testing
**URL to test:** 

local because it consists of database changes

**Steps to test:**

1. add a new phase to a project in moped
2. check the record in the `moped_proj_phases` table to confirm the `updated_at` timestamp matches the `created_at` timestamp by checking the last four decimal places
3. go to that record's parent project and confirm the `updated_at` timestamp matches its child record's timestamp fields
4. edit the phase you just added in moped by updating the description
5. confirm that both the record and its parent record's `updated_at` timestamps updated accordingly
6. repeat these steps for `moped_proj_milestones`, `moped_proj_funding`, `moped_proj_work_activity` (on the funding tab), `moped_proj_notes`, and `moped_proj_components`

---
#### Ship list
